frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -88,12 +89,23 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
               defaultRequestSizeUnit={requestSizeUnit}
               defaultRequestSizeValue={requestSizeValue}
               dropdownUnits={dropdownUnits}
+              step={2}
+              minValue={2}
+              inputClassName="add-capacity-modal__input--width"
               required
             />
+            <p
+              className={classNames('text-secondary add-capacity-modal__help-text', {
+                'add-capacity-modal__help-text-error': buttonDisabled,
+              })}
+            >
+              Please enter an even number
+            </p>
           </div>
-          <div className="toolTip_dropdown">
+          <label className="control-label">
```

Comment:
This bit appears to be slightly different as well.

Reply:
ack